src/configs/common/v1-base-config

```diff
@@ -16,3 +16,5 @@ opt_enable SHOW_CUSTOM_BOOTSCREEN
 echo "# ${CONFIG_NAME} - ${MARLIN_VERSION} - ${V1_VERSION}" > README.md
 cat $CFGDIR/common/Readme_header.md >> README.md
```

Comment:
I believe the current coding style is for the lines that update README.md to come last. @jeffeb3 can comment if he had an opinion here of am I just trying to read his mind ;)

Reply:
That could be, let me try to change it.

Reply:
You can call it style, but I think it helps to make it consistent. :+1:

Reply:
Did I fix it right?